CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Add IBAN validation in frontend and backend
 - Remove IBAN and accountHolderNumber if refund method different than bank
+- Stop relying on saving the sellerName at the creation of a return request. Instead get the seller name when resolving the objects that serve it to the front end.
```

Comment:
Please rebase and make sure changelog is right. The changes listed above are already released.